NetKAN/SpaceDock.netkan

```diff
@@ -0,0 +1,38 @@
+ 
+{
+    "spec_version"   : "v1.20",
+    "name"           : "SpaceDock",
+    "abstract"       : "SpaceDock in different sizes, usable as orbital shipyard.",
+    "author"         : "Fendrin",
+    "identifier"     : "SpaceDock",
+    "download"       : "https://github.com/fendrin/SpaceDock/archive/v0.2.0.zip",
+    "license"        : "CC-BY-NC-SA-4.0",
+    "version"        : "0.2.0",
+    "release_status" : "stable",
+    "ksp_version"    : "1.2.0",
+    "resources" : {
+        "homepage"     : "http://forum.kerbalspaceprogram.com/index.php?/topic/150589-12-spacedock",
+        "repository"   : "https://github.com/fendrin/SpaceDock"
+    },
+    "install" : [
+        {
+            "file"       : "SpaceDock-0.2.0/GameData/SpaceDock",
```

Comment:
Make this 

    "find" : "GameData/SpaceDock",

And it won't need to be changed for every new release.